Source/WebKit/UIProcess/API/glib/WebKitUserContentManager.cpp

```diff
@@ -224,12 +261,140 @@ void webkit_user_content_manager_remove_all_scripts(WebKitUserContentManager* ma
     manager->priv->userContentController->removeAllUserScripts();
 }
 
+/**
+ * WebKitScriptMessageReply: (ref-func webkit_script_message_reply_ref) (unref-func webkit_script_message_reply_unref)
+ *
+ * A reply for a script message received.
+ * If no reply has been sent by the user, an automatically generated reply with
+ * undefined value with be sent.
+ *
+ * Since: 2.40
+ */
+struct _WebKitScriptMessageReply {
+    _WebKitScriptMessageReply(WTF::Function<void(API::SerializedScriptValue*, const String&)>&& completionHandler)
+        : completionHandler(WTFMove(completionHandler))
+        , referenceCount(1)
+    {
+    }
+
+    void sendValue(JSCValue* value)
+    {
+        JSCContext* jscContext = SharedJavascriptContext::singleton().getOrCreateContext();
+        auto serializedValue = API::SerializedScriptValue::create(jscContextGetJSContext(jscContext), jscValueGetJSValue(value), nullptr);
+
+        completionHandler(serializedValue.get(), { });
+    }
+
+    void sendErrorMessage(const char* errorMessage)
+    {
+        completionHandler(nullptr, String::fromUTF8(errorMessage));
+    }
+
+    ~_WebKitScriptMessageReply()
+    {
+        if (completionHandler) {
+            auto value = adoptGRef(jsc_value_new_undefined(SharedJavascriptContext::singleton().getOrCreateContext()));
```

Comment:
Same here, after commit https://github.com/WebKit/WebKit/commit/4041ec7696c53b8f7e64d3046d4f7ae3d180cf35 this can be:

```suggestion
            auto value = adoptGRef(jsc_value_new_undefined(API::SerializedScriptValue::sharedJSCContext()));
```